### PR TITLE
Fixed IHP Template being broken

### PR DESCRIPTION
**Pull request by mpscholten**
## Description
The ihp template is somehow currently broken with git saying `fatal: Cannot rebase onto multiple branches.`.

## Related Issue(s)
This problem has been first reported here: https://ihp.digitallyinduced.com/community/ShowThread?threadId=a463718b-d427-401e-a168-fa541e7b3a5b

### CODE REVIEW

**Comment by mpscholten** [comment]
hey @ghuntley 👋 , can you take a look?

File: .gitpod.yml [hunk 1]
```diff
@@ -8,13 +8,10 @@ ports:
   # IDE
   - port: 8001
     onOpen: open-browser
-  # Live Reloading
```
Comment (amitaibu) [inline .gitpod.yml:11]: I think the first port should be `8000` instead of `8001`?

Comment (mpscholten) [reply .gitpod.yml:11]: Really good catch, thanks :) I'll update that 👍

File: .gitpod.yml [hunk 2]
```diff
@@ -3,18 +3,15 @@ image:
 
 ports:
   # IHP Server
-  - port: 8001
+  - port: 8000
```
Comment (amitaibu) [inline .gitpod.yml:6]: Let's also add `visibility: public`, so one could `curl` into it and get a response

Comment (mpscholten) [reply .gitpod.yml:6]: I'd like to keep `private` the default, as that's more secure for most users (e.g. you might have sensitive data inside your fixtures that you don't want to expose to the internet). You can still make a project public using the UI if needed

**Pull request merged**